new-colorlib/style.css

```diff
@@ -102,7 +101,6 @@ input {
   background-size: cover;
   color: white;
   position: relative;
-  -webkit-backdrop-filter: blur(8px);
 }
 .grid-right {
```

Comment:
- [ ] For your images, adjust your CSS to look something like this;
```
.grid-sm1 {
  background: url(../new-colorlib/img/2.jpg) no-repeat center center/cover;
  position: relative;
  width: 100%;
  height: 100%;
}
```

```
.grid-left-icon {
  position: absolute;
  width: 100%;
  height: 100%;
  background: rgba(0, 0, 0, 0.5); /* Dim effect */
  display: flex;
  align-items: flex-end;
  color: #f3f0f0;
}
```

- [ ] Read more on how to add background images